ddtrace/sampler.py

```diff
@@ -109,30 +115,40 @@ def set_sample_rate_by_service(self, rate_by_service):
 RateByServiceSampler._default_key = RateByServiceSampler._key()
 
 
-class DatadogSampler(BaseSampler):
+class DatadogSampler(BaseSampler, BasePrioritySampler):
     """
     This sampler is currently in ALPHA and it's API may change at any time, use at your own risk.
     """
     __slots__ = ('default_sampler', 'limiter', 'rules')
 
     NO_RATE_LIMIT = -1
     DEFAULT_RATE_LIMIT = 100
-    DEFAULT_SAMPLE_RATE = 1.0
+    DEFAULT_SAMPLE_RATE = None
 
     def __init__(self, rules=None, default_sample_rate=None, rate_limit=None):
         """
         Constructor for DatadogSampler sampler
 
         :param rules: List of :class:`SamplingRule` rules to apply to the root span of every trace, default no rules
         :type rules: :obj:`list` of :class:`SamplingRule`
-        :param default_sample_rate: The default sample rate to apply if no rules matched (default: 1.0)
+        :param default_sample_rate: The default sample rate to apply if no rules matched (default: ``1.0``)
```

Comment:
The default is wrong as it's disabled by default

Reply:
```suggestion
        :param default_sample_rate: The default sample rate to apply if no rules matched (default: ``None``/Use :class:`RateByServiceSampler` only)
```